.github/workflows/continuous-integration.yml

```
@@ -73,8 +73,10 @@ jobs:
             exit 0
           fi
 
+          pip install osu-wiki-tools
+
```

you have installed it on the line 63 already
```suggestion
          pip install osu-wiki-tools

```